ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -45,6 +51,54 @@ public String getName() {
     return "ImageLoader";
   }
 
+  @ReactMethod
+  public void getSize(
+      String uriString,
+      final Promise promise) {
+    Uri uri = Uri.parse(uriString);
```

Comment:
Add the same check that `prefetch` does above this. Uri.parse will throw if it gets null, and we want to always either resolve or reject the promise.